### PR TITLE
Update pt-PT.json

### PR DESCRIPTION
**Pull request by smarquespt**

### CODE REVIEW

**Comment by PalmerAL** [comment]
Thanks!

Would you be willing to add the strings for [password autofill](https://github.com/minbrowser/min/blob/master/localization/languages/en-US.json#L205-L227) as well?

**Comment by smarquespt** [comment]
Sure. But I don´t know how to squash commits. So please merge this one and I will add other strings later

**Comment by PalmerAL** [comment]
It's fine, Github has an option to squash it automatically when I merge it.

**Comment by smarquespt** [comment]
Please check https://github.com/minbrowser/min/pull/933

File: localization/languages/pt-PT.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
         /* Context menu items
         these are the items displayed in the menu when you right-click on a page
         */
-        "pictureInPicture": null, //missing translation
+        "pictureInPicture": "Picture in Picture",
```
Comment (PalmerAL) [inline localization/languages/pt-PT.json]: Is leaving this in English intentional?

Comment (PalmerAL) [reply localization/languages/pt-PT.json]: This was answered in #937.

**Comment by PalmerAL** [comment]
Thank you!

**Pull request merged**